addons/website_blog/static/src/scss/website_blog.scss

```diff
@@ -238,11 +207,11 @@ $o-wblog-title-sizes-variants: (
                     visibility: hidden;
                 }
 
-                &.cover_full {
+                &.cover_full, &.o_full_screen_height {
```

Comment:
In fact, the css could be better. What you have now is new blog posts with the new classes which have this rule:
`.o_full_screen_height { hello }`
and old blog posts with old classes which have this rule:
`.a . b . c .d .e .cover_full { hello }`
-> the same one with a different priority.

If the blogs are still working with the low priority rule, then better to make the compatibility scss this way:
1. Replace all occurrences of old classes in the code by the new class names (so here, only `o_full_screen_height`)
2. Remove the rules which are duplicate of the base rule for full and half (where you added the `@extend` in your current implementation)
3. In website, below the definition of the new classes, do:
```scss
// TODO compatibility blablabla
.cover_full {
    @extend .o_full_screen_height;
}
// ...